Mathlib/Algebra/Order/Group/Abs.lean

```diff
@@ -56,21 +56,25 @@ variable [LinearOrderedCommGroup α] {a b : α}
   · exact mabs_mul_eq_mul_mabs_le ab
   · simpa only [mul_comm, and_comm] using mabs_mul_eq_mul_mabs_le ab
 
-end LinearOrderedCommGroup
+@[to_additive]
+theorem mabs_le : |a|ₘ ≤ b ↔ b⁻¹ ≤ a ∧ a ≤ b := by rw [mabs_le', and_comm, inv_le']
```

Comment:
I added multiplicative versions of these lemmas, because one of them is used in the `AtTopBot/Group` file. We should do the same to the whole section below, but I consider this to be out of the scope of this PR.